src/types/index.tsx

```diff
@@ -42,6 +42,36 @@ export type UserDetails = {
   status: UserStatus;
 };
 
+export interface LeaveRequestData {
+  memberId: string;
+  leaveType: string;
+  startDate: Date;
+  endDate: Date;
+  reasonForLeave: string;
+  status: string;
+}
+export interface MembersProps {
+  data: Member[];
+  success: boolean;
+  message: string;
+}
+
+export interface LeaveRequestsTypes {
```

Comment:
We need these types to be localized. If we don't intend to reuse them, they should be close to consumer.